src/repositories/iflow_node_instance_repository.ts

```diff
@@ -91,4 +91,16 @@ export interface IFlowNodeInstanceRepository {
     offset?: number,
     limit?: number,
   ): Promise<Array<FlowNodeInstance>>;
+
+  /**
+   * Resets FlowNodeInstance's state to running and deletes its token except the onEnter token.
+   *
+   * This only works on FlowNodeInstances, which have previously been put
+   * into a 'error' state.
```

Comment:
Solche linebreaks bitte vermeiden. Dieses Nur-80-Zeichen-Pro-Zeile-Dogma gibts nicht mehr. :wink: 

```suggestion 
   * Works only on FlowNodeInstances that are in an error state.
```